net-misc/sipp/sipp-3.6.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="SIPp is a free Open Source test tool / traffic generator for the SIP protocol"
+HOMEPAGE="http://sipp.sourceforge.net/ https://github.com/SIPp/sipp/releases"
+SRC_URI="https://github.com/SIPp/sipp/releases/download/v${PV}/${PF}.tar.gz"
+
+LICENSE="GPL-2 ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl +pcap sctp +ssl"
+
+DEPEND="sys-libs/ncurses:=
+	ssl? ( dev-libs/openssl:= )
+	pcap? ( net-libs/libpcap net-libs/libnet:1.1 )
+	gsl? ( sci-libs/gsl:= )
+"
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	econf \
+		$(use_with gsl) \
+		$(use_with pcap) \
+		$(use_with sctp) \
+		$(use_with ssl openssl) \
+		--with-rtpstream
+}
+
+src_install() {
+	default
+	insinto /usr/share/${PN}
+	doins pcap/*.pcap
+	dodoc CHANGES.md FAQ.md README.md
```

Comment:
```
>>> Install net-misc/sipp-3.6.0 into /tmp/portage/net-misc/sipp-3.6.0/image
make -j16 -l16 DESTDIR=/tmp/portage/net-misc/sipp-3.6.0/image install 
make  install-am
make[1]: Entering directory '/tmp/portage/net-misc/sipp-3.6.0/work/sipp-3.6.0'
make[2]: Entering directory '/tmp/portage/net-misc/sipp-3.6.0/work/sipp-3.6.0'
 /bin/mkdir -p '/tmp/portage/net-misc/sipp-3.6.0/image/usr/bin'
 /bin/mkdir -p '/tmp/portage/net-misc/sipp-3.6.0/image/usr/share/man/man1'
  /usr/lib/portage/python3.6/ebuild-helpers/xattr/install -c sipp '/tmp/portage/net-misc/sipp-3.6.0/image/usr/bin'
 /usr/lib/portage/python3.6/ebuild-helpers/xattr/install -c -m 644 sipp.1 '/tmp/portage/net-misc/sipp-3.6.0/image/usr/share/man/man1'
make[2]: Leaving directory '/tmp/portage/net-misc/sipp-3.6.0/work/sipp-3.6.0'
make[1]: Leaving directory '/tmp/portage/net-misc/sipp-3.6.0/work/sipp-3.6.0'
Traceback (most recent call last):
  File "/usr/lib/portage/python3.6/doins.py", line 611, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/usr/lib/portage/python3.6/doins.py", line 602, in main
    os.path.dirname(source)):
  File "/usr/lib/portage/python3.6/doins.py", line 450, in _doins
    return install_runner.install_file(source, os.path.dirname(dest))
  File "/usr/lib/portage/python3.6/doins.py", line 386, in install_file
    return self._ins_runner.run(source, dest_dir)
  File "/usr/lib/portage/python3.6/doins.py", line 195, in run
    sstat = os.stat(source)
FileNotFoundError: [Errno 2] No such file or directory: b'FAQ.md'
 * ERROR: net-misc/sipp-3.6.0::testworld failed (install phase):
 *   dodoc failed
```
:(

Reply:
Oh, and:
 * QA Notice: make jobserver unavailable:
 * 
 * 	make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.

If you can patch it, good. If not, make it clear in the ebuild this won't build parallel.

Reply:
Not sure how I missed FAQ.md no longer being there.

Also made the install of the pcap files dependent on USE=pcap since that's required to be able to replay the pcap files (DTMF, highly recommended) anyway.

I believe that the Makefile should be adequately patched.  It only affects a "directly after ./configure forced clean" as far as I can determine, however, please confirm.  Not sure how you're getting that QA notice (would appreciate if you can share the magic, google isn't friendly with me ...).